app/webpacker/components/RegistrationsV2/Registrations/RegistrationList.jsx

```diff
@@ -94,8 +94,10 @@ export default function RegistrationList({ competitionInfo }) {
   const { sortColumn, sortDirection } = state;
   const changeSortColumn = (name) => dispatch({ type: 'CHANGE_SORT', sortColumn: name });
 
+  const blindFoldedEvents = ['333mbf', '333bf', '444bf', '555bf'];
```

Comment:
You don't need to hardcode this. We have events data available as `import { events } from 'lib/wca-data.js.erb'` (note you might need to change the relative path to the file)